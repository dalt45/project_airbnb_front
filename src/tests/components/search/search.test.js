import React from 'react';
import { shallow } from'enzyme';
import SearchForm from '../../../components/search/search';

test('should render SearchForm page correctly', () => {
   const wrapper = shallow(<SearchForm />);
   wrapper.find('TextField').at(0).simulate('change',{
       target:{
           value:"guadalajara"
       }
   })
   expect(wrapper.state('search')).toEqual("guadalajara")
   expect(wrapper).toMatchSnapshot();
})

