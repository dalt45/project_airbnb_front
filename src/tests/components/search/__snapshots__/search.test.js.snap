// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchForm page correctly 1`] = `
<div>
  <WithStyles(Card)>
    <TextField
      id="city"
      onChange={[Function]}
      required={false}
      select={false}
      value="guadalajara"
    />
    <WithStyles(Button)
      color="secondary"
      onClick={[Function]}
      variant="contained"
    >
      Buscar
    </WithStyles(Button)>
  </WithStyles(Card)>
</div>
`;
